### PR TITLE
Enabling multiple kme wpe CI

### PR DESCRIPTION
**Pull request by jinengandhi-intel**

### CODE REVIEW

File: tests/ci/avalon-multiple-wp-direct-model-sgx-tests.yaml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      - 1947
+      - 5555
+    command: |
+      bash -c "
+        avalon_listener --bind http://avalon-listener:1947 --lmdb_url http://avalon-lmdb:9090
+        tail -f /dev/null
+      "
+    depends_on:
+      - avalon-lmdb
+
```
Comment (pankajgoyal2) [inline tests/ci/avalon-multiple-wp-direct-model-sgx-tests.yaml:179]: This file has single instance of wpe and kme?

File: tests/ci/avalon-multiple-wp-direct-model-tests.yaml [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    environment:
+      - http_proxy
+      - https_proxy
+      - no_proxy
+    expose:
+      - 1949
+      - 5555
+    command: |
+      bash -c "
+        enclave_manager --lmdb_url http://avalon-lmdb:9090 --bind http://avalon-kme-2:1949 --worker_id worker_pool2
```
Comment (pankajgoyal2) [inline tests/ci/avalon-multiple-wp-direct-model-tests.yaml]: No need to change port from 1948 to 1949 for avalon-kme-2. You can use the same port 1948.

File: tests/ci/avalon-multiple-wp-direct-model-tests.yaml [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+    image: avalon-enclave-manager-wpe:${ISOLATION_ID}
+    environment:
+      - http_proxy
+      - https_proxy
+      - no_proxy
+    expose:
+      - 5555
+    command: |
+      bash -c "
+        enclave_manager --lmdb_url http://avalon-lmdb:9090 --kme_listener_url http://avalon-kme-2:1949 --worker_id worker_pool2
```
Comment (pankajgoyal2) [inline tests/ci/avalon-multiple-wp-direct-model-tests.yaml]: No need to change port from 1948 to 1949 for avalon-kme-2. You can use the same port 1948.

File: tests/ci/run_tests [hunk 4]
```diff
@@ -76,11 +78,15 @@ main() {
         run_docker_test ./ci/avalon-ganache-proxy-model-wp-tests.yaml
     elif [ "$GANACHE_PROXY_MODEL_WP_SGX_TESTS" == "1" ]; then
         run_docker_test ./ci/avalon-ganache-proxy-model-wp-sgx-tests.yaml
+	elif [ "$MULTIPLE_WP_DIRECT_MODEL_TESTS" == "1" ]; then
+        run_docker_test ./ci/avalon-multiple-wp-direct-model-tests.yaml
+	elif [ "$MULTIPLE_WP_DIRECT_MODEL_SGX_TESTS" == "1" ]; then
+        run_docker_test ./ci/avalon-multiple-wp-direct-model-sgx-tests.yaml
```
Comment (pankajgoyal2) [inline tests/ci/run_tests:84]: please align the newly added lines to previous lines. Remove extra spaces added at the beginning.

File: tests/ci/run_tests [hunk 5]
```diff
@@ -130,6 +136,12 @@ while getopts "dbfgwstuvxyzijkl" opt; do
         l)
             GANACHE_PROXY_MODEL_WP_SGX_TESTS=1
             ;;
+		a)
+			MULTIPLE_WP_DIRECT_MODEL_TESTS=1
+			;;
+		c)
+			MULTIPLE_WP_DIRECT_MODEL_SGX_TESTS=1
+			;;
```
Comment (pankajgoyal2) [inline tests/ci/run_tests:144]: please align the newly added lines to previous lines. Remove extra spaces added at the beginning.

**Review by pankajgoyal2 — CHANGES_REQUESTED** [review]

File: tests/ci/avalon-multiple-wp-direct-model-sgx-tests.yaml (hunk 1 above)
Comment (jinengandhi-intel) [reply tests/ci/avalon-multiple-wp-direct-model-sgx-tests.yaml:179]: It has 2 KME, 1 having just 1 WPE and 1 having 2 WPE.

Comment (jinengandhi-intel) [reply tests/ci/avalon-multiple-wp-direct-model-sgx-tests.yaml:179]: Addressed.

**Review by jinengandhi-intel — COMMENTED** [review]
Addressed comments on ports and KME/WPE.

**Review by pankajgoyal2 — APPROVED** [review]

**Pull request merged**